Makefile

```
@@ -91,20 +93,29 @@ upload: check_upkie_name build  ## upload built targets to the Raspberry Pi
 		--exclude logs/ppo \
 		--exclude logs/tensorboard \
 		--exclude tools/bazel \
+		--exclude tools/raspios \
 		--progress $(CURDIR)/ ${UPKIE_NAME}:$(PROJECT_NAME)/
 
 # REMOTE TARGETS
 # ==============
-
 run_mpc_balancer:  ### run agent
-	@if [ -f $(CURDIR)/activate.sh ]; then \
-		echo "Running MPC balancer from packed environment..."; \
-		. $(CURDIR)/activate.sh && python -m agents.mpc_balancer; \
+	@if [ -f /home/pi/micromamba/envs/activate.sh ]; then \
```

Oh, that's right, when we unpack we are putting the activation script into the mamba root prefix as well... It's actually a bit dirty :blush: if other projects do this, `activate.sh` will be overwritten. Hmm.

Let's use `MAMBA_ROOT_PREFIX` for consistency for now?

```suggestion
	@if [ -f ${MAMBA_ROOT_PREFIX}/envs/activate.sh ]; then \
```